plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/runner/statement/AbstractStatementWithScreenshot.java

```diff
@@ -104,6 +105,16 @@ protected void createScreenshot(String description, Class<?> extensionClass) {
 		}
 	}
 	
+	/**
+	 * Tests if screenshot is required on throwable.
+	 * 
+	 * @param throwable the throwable
+	 * @return true if required, false otherwise
+	 */
+	protected boolean isScreenshotRequired(Throwable throwable){
+		return !(throwable instanceof AssumptionViolatedException);
```

Comment:
use check of exception type in Requirements class as well - there is a logic in canFulfill and fulfil methods where exception is catched and screenshot captured. We should have it consistent.